draft-birkholz-rats-corim.md

```diff
@@ -99,6 +99,52 @@ See {{-coswid}}.
 
 Privacy Considerations
 
+# Concise SWID Data Definition
+
+# 2.7 The link-entry Map
+
+The link-entry map has been extended to define new relationship between a CoSWID and
+CoMID(identifies a hardware/firmware module).
+
+When defining CoSWID to CoMID link entry map the following optional elements
+should not be used:
+ artifact
+ ownership
+ use
+
+# CoSWID Indexed Label Values
+
+# 4.4. Link Rel Values
```

Comment:
Is this further section really needed?  I think this should be in the context of the section above (i.e., "Link-entry Map")

Reply:
This is how it is documented in CoSWID specification in an exact manner! Hence I replicated the tabular syntax from
Section 4.4 of CoSWID specification!